docs/university/core-components/collections.md

```diff
@@ -0,0 +1,44 @@
+# 🗄️ Collections
+In Webstudio, "Collections" are a powerful feature designed to manage and display dynamic content on web pages. Essentially, they serve as containers that can connect to various data sources, like databases or APIs, to fetch and present data in a structured and repeatable manner. This allows developers and designers to create elements, such as lists, galleries, or testimonial sections, where each item in the collection is automatically populated with data from the connected source. By using Collections, Webstudio users can efficiently create pages with content that updates in real-time, reflecting changes in the data source without manual intervention, thereby enhancing the site's interactivity and relevance.
```

Comment:
"create elements" we refer generally to things on canvas as "instances", its a bit less popular but more correct, because Components are like a class, when you create something on canvas you create an instance of a component, an instance of a component may or may not have have dom elements. I don't want to cause confusion in our language, so we need a consistent use of terminology 🙏

Reply:
A lot of text in this paragraph is a bit markety e.g. "powerful" or vague e.g. "manual intervention" - not clear what that means, "enhancing the site's interactivity and relevance" also a questionable choice of words or points to make. If you are trying to explain what it means to have dynamic data on the site, perhaps it could be a separate paragraph that educates user and provides examples. Its a bit weird to see a mixture of explanation for webstudio collections with all the contextual information that may be not interesting or already known to the user.

I would strongly suggest we need to separate what's specific to webstudio and what's common knowledge 🙏